diceRoll/diceRollUnitTests.js

```diff
@@ -65,63 +38,118 @@ function assert(condition, message) {
   }
 }
 
+function stub() {
+  var info = {
+    called: {
+      true: false,
+      expectedCount: null,
+      count: 0,
+      check: function(expectedCount) {
+        info.called.expectedCount = expectedCount;
```

Comment:
Do you need to store the expected count or just compare it?

(See my comment below about info.called count) then you can boil all of check fct down to:

return expectedCount && expectedCount === info.called || info.called > 0;

Reply:
return expectedCount && expectedCount === info.called.count || info.called.count > 0;